nutils/evaluable.py

```diff
@@ -2030,7 +2030,7 @@ class Take(Array):
 
     def __post_init__(self):
         assert isinstance(self.func, Array) and self.func.ndim > 0, f'func={self.func!r}'
-        assert isinstance(self.indices, Array) and self.indices.dtype == int, f'indices={self.indices!r}'
+        assert isinstance(self.indices, Array) and self.indices.dtype == int and self.indices._intbounds[0] >= 0, f'indices={self.indices!r}'
```

Comment:
I'm a bit afraid of this assertion, so close to the release. What if we missed something? Could we restrict the next commit to positive scalars (I imagine they're typically constant?) and leave this one for after the release?

Reply:
I'm also a bit hesitant. Let's postpone this.